docs/experimental/viz_points_kde.py

```diff
@@ -0,0 +1,277 @@
+import numpy as np
+from fury import window, actor
+from fury.shaders import compose_shader, shader_apply_effects
+from fury.lib import Texture, WindowToImageFilter
+from fury.io import load_image
+from fury.utils import rgb_to_vtk
+from matplotlib import colormaps
+
+
+def window_to_texture(
+        window : window.RenderWindow,
+        texture_name : str,
+        target_actor : actor.Actor,
+        blending_mode : str = "None",
+        wrap_mode : str = "ClampToBorder",
+        border_color : tuple = (
+            0.0,
+            0.0,
+            0.0,
+            1.0),
+        interpolate : bool = True):
+    """Captures a rendered window and pass it as a texture to the given actor.
+
+    Parameters
+    ----------
+    window : window.RenderWindow
+        Window to be captured.
+    texture_name : str
+        Name of the texture to be passed to the actor.
+    target_actor : actor.Actor
+        Target actor to receive the texture.
+    blending_mode : str, optional
+        Texture blending mode. The options are:
+    1. None
+    2. Replace
+    3. Modulate
+    4. Add
+    5. AddSigned
+    6. Interpolate
+    7. Subtract
+
+    wrap_mode : str, optional
+        Texture wrapping mode. The options are:
+    1. ClampToEdge
+    2. Repeat
+    3. MirroredRepeat
+    4. ClampToBorder
+
+    border_color : tuple (4, ), optional
+        Texture RGBA border color.
+    interpolate : bool, optional
+        Texture interpolation."""
+
+    wrap_mode_dic = {"ClampToEdge" : Texture.ClampToEdge,
+                     "Repeat" : Texture.Repeat,
+                     "MirroredRepeat" : Texture.MirroredRepeat,
+                     "ClampToBorder" : Texture.ClampToBorder}
+
+    blending_mode_dic = {"None" : 0, "Replace" : 1,
+                         "Modulate" : 2, "Add" : 3,
+                         "AddSigned" : 4, "Interpolate" : 5,
+                         "Subtract" : 6}
```

Comment:
I set those like that because it is how their names are defined inside VTK, however this suggestion makes sense as a way to prevent this kind of problems a user may have. I think it is an easy and minor thing to be modified, so for me that seems alright to do 👍